.travis.yml

```diff
@@ -21,7 +21,11 @@ before_install:
     - ./configure && make
     - sudo make install
     - cd ..
-    - git pull --recurse-submodules && git submodule update
+    - git config user.email "travis@example.com"
+    - git config user.name "Travis"
+    - if [[ -d viper/modules/pdftools ]]; then rm -rf viper/modules/pdftools ; fi
+    - git pull --recurse-submodules --no-commit && git submodule update
+
```

Comment:
I had to add these four lines to get Travis to run.. should you accept to merge this, then you can clean this up...